spec/api-browser-window-spec.js

```diff
@@ -1185,7 +1185,7 @@ describe('BrowserWindow module', function () {
         })
       })
 
-      it('can print to PDF', function (done) {
+      xit('can print to PDF', function (done) {
```

Comment:
This should be noted somewhere that it was disabled and needs to be re-enabled before `1.8.x` can go stable